tests/cross/src/test/scala/tests/pc/AutoImportsSuite.scala

```diff
@@ -220,6 +234,19 @@ class AutoImportsSuite extends BaseAutoImportsSuite {
       )
   )
 
+  checkAmmoniteEdit(
```

Comment:
Let's remove this and ignore the ammonite tests for Scala 3 since we don't really support it.